src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js

```diff
@@ -27,8 +26,7 @@ export default class GeGiftEntryFormApp extends LightningElement {
                 }
             )
             .catch(error => {
-                handleError(error);
```

Comment:
What's the consequence of not calling the utilTemplateBuilder handle error here? Just want to make sure we don't have a broader, unintentional change to error handling behavior

Reply:
That method was causing an issue since the error was not in the expected format

Reply:
Worth to mention that the issue existed before I changed the exception handling on BGE_DataImportBatchEntry_CTRL class